src/libImaging/ImPlatform.h

```diff
@@ -61,17 +83,22 @@
 #if SIZEOF_LONG == 8
 #define INT64 long
 #elif SIZEOF_LONG_LONG == 8
-#define INT64 long
+#define INT64 long long
+#else
+#warning Cannot find required 64-bit integer type
 #endif
 
-#define INT8 signed char
 #define UINT8 unsigned char
-
 #define UINT16 unsigned INT16
 #define UINT32 unsigned INT32
-
+#ifdef INT64
+#define UINT64 unsigned INT64
```

Comment:
But you have added UINT64, which also doesn't seem to be used anywhere?

Reply:
If there's one I figure we might as well have the other.

Reply:
After #7180 was merged, INT64 is no longer present, so I've created https://github.com/Yay295/Pillow/pull/7 to remove UINT64.